### PR TITLE
Added descriptions and examples in R docs

### PR DESCRIPTION
**Pull request by jameslamb**
Hey friends! In this PR, I propose some upgrades to the roxygen docs for this project, including descriptions and examples.

I also explicitly put `@export` tags on `KafkaBroker`, `KafkaConsumer`, and `KafkaProducer`. Your examples in the README suggest that you want them exported.

### CODE REVIEW

**Comment by jameslamb** [comment]
partially contributes towards #8

File: README.md [hunk 1]
```diff
@@ -1,11 +1,13 @@
 # fRranz: An R Kafka Client
+
+![Lifecycle badge](https://img.shields.io/badge/lifecycle-experimental-orange.svg)
```
Comment (jameslamb) [inline README.md:3]: https://www.tidyverse.org/lifecycle/

**Review by mjermann — DISMISSED** [review]
Fantastic examples!

**Comment by mjermann** [comment]
digging into the travis build issues!

**Review by Paxanator — APPROVED** [review]
thanks @jameslamb

**Pull request merged**